index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <script src="https://kit.fontawesome.com/7b47055909.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="./style.css" type="text/css">
+    <link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:300&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Raleway:200&display=swap" rel="stylesheet">
+    <title>Document</title>
+</head>
+<body>
+<!-- <div class="container"> -->
+    <div class="back-ground-image">
+        <div class="nav">
+        
+        <div  class="nav-tabs"><a href="#"><i id="apple" class="fab fa-apple fa-2x"></i></a></div>
+        <div class="nav-tabs"><a href="">Store</a></div>
+        <div class="nav-tabs"><a href="">Mac</a></div>
+        <div class="nav-tabs"><a href="">iPod</a></div>
+        <div class="nav-tabs"><a href="">iPhone</a></div>
+        <div class="nav-tabs"><a href="">iPad</a></div>
+        <div class="nav-tabs"><a href="">iTunes</a></div>
+        <div class="nav-tabs"><a href="">Support</a></div>
+        <!-- <div class="search"> -->
+            <i class="fas fa-search fa-xs"></i>
+
+           <div class="nav-tabs"><form action="post"><input type="search"></form></div>
+            
+        <!-- </div> -->
+    
+        </div>
+        <div class="slogans">
+            <a href="#"><h2 id="verse">What will your verse be?</h2></a>
+        <a href="#"><h2>Everyone has something to share. See how poeple are using <br><br> iPad in their unique ways to add to the world's story.</h2></a>
+        <div class="last-slogan">
+        <a href="#"><h3>Watch the "your verse" film and explore the stories<i id="play" class="far fa-play-circle"></i></h3>
+            </a>
+            </div>
+        </div>
+    </div>
+    <div class="bottom-images">
+    <img src="https://archive.fo/UW4oR/4199f096e33e638ba9aeb6c6dec3a3e64794d02c.jpg" alt="">
+    <img src="https://archive.fo/UW4oR/6032fe46fd16493ddda4e90ae45725962fc826b9.jpg" alt="">
+    <img src="https://archive.fo/UW4oR/307a4ef1971d63d523c362cc3e49f923549fcefc.jpg" alt="">
+    <img src="https://archive.fo/UW4oR/ec67ffeb084d1b9c83687205641697baa4122bee.jpg" alt="">

```

Comment:
These should be links as can be seen from the original. They can either link to articles or `#`

Reply:
When you hover it should be obvious it is a link

Reply:
Please wrap each `img` in an anchor tag to make them links.